src/groupByKey.ts

```diff
@@ -1,7 +1,20 @@
+
 type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey<T>(items: T[], key: keyof T): GroupsMap<T> {
+  const GroupsMap: GroupsMap<T> = {};
```

Comment:
```suggestion
  const groupsMap: GroupsMap<T> = {};
```
do not name variables with upperCase !!!!!!!!!!